portfolio/src/main/webapp/index.html

```diff
@@ -1,185 +1,1224 @@
+                <a title="file size: {NoteCount} B" id="openpost11">
+                <a title="file size: {NoteCount} B" id="openpost12">
+                <a title="file size: {NoteCount} B" id="openpost13">
+                <a title="file size: {NoteCount} B" id="openpost14">
+                <a title="file size: {NoteCount} B" id="openpost15">
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-font"></i></div> .txt</div></div><!--postnav-->
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-image"></i></div> img.jpg</div></div><!--postnav-->
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-th-large"></i></div> photoset</div></div><!--postnav-->
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-fast-forward"></i></div> vid.mp4</div></div><!--postnav-->
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-link"></i></div></div></div><!--postnav-->
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-music"></i></div> track.mp3</div></div>
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-comment-o"></i></div> chat log 03/21</div></div>
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-quote-left"></i></div> quote.txt</div></div>
+                <div class="postnav"><div class="pnex"><div class="posticon"><i class="fa fa-envelope-o"></i></div> mail received on 03/06/21</div></div>
+                <div class="permapostinfo">
+                <blockquote>
+                posted: n/a <br>
+                notes: note count
+                </blockquote>
+                RebloggedFrom yukoki
+                <blockquote>
+                via <a href="https://google.com">Google Software Product Sprint</a>
+                <br>originally <a href="https://google.com">Google</a>
+                </blockquote>
+                <blockquote>
+                <a href="{TagURL}"></a>
+                </blockquote>
+                </blockquote>
+                <br><a href="/"><i class="fa fa-angle-double-left"></i> back to main</a>
+                </div><!--permapostinfo-->
+                </a><!-- openpost x end link -->
+                <div style="height:10px;"></div>
+                <a class="jump_page" href="https://google.com"><div class="inactivefolder"><span style="margin-right:6px;"><i class="fa fa-caret-right"></i></span> folder </div></a>
+                <div class="totalfoldercount">total folders</div>
+                </div><!--folders-->
+                
+                </div><!--contentleft-->
+                
+                
+                </div><!-- contentpop -->
+
+
+                 
+                <div id="sticky1" class="ui-draggable">
```

Comment:
I see a lot of content on this page. Are you considering personalizing it or leaving it as is?

Reply:
Yes, I want to make all the icons draggable and personalize the stickys